packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/phone.js

```diff
@@ -219,7 +198,7 @@ browserOnly(describe)(`plugin-phone`, function() {
           }));
 
       // TODO [SSDK-233, SSDK-508, SSDK-56, SSDK-149, SSDK-3] need to figure out
-      // what entitlements are required to dial sip addresses. Also, might be
+      // what entitlements are required to a dial sip addresses. Also, might be
```

Comment:
Very minor but probably meant to be `to dial a sip`

Reply:
will fix in next PR